JSTests/wasm/stress/cc-int-to-int-cross-module.js

```diff
@@ -39,7 +37,7 @@ async function test() {
     const instance2 = await instantiate(wat2, { o: { test, tbl } }, { simd: true })
     const { test_with_call, test_with_call_indirect } = instance2.exports
 
-    for (let i = 0; i < 10000000; ++i) {
+    for (let i = 0; i < 1000; ++i) {
```

Comment:
If I understand correctly, you've changed a good number of the tests which previously used `@requireOptions("--useJITCage=0")` as a workaround for using too much memory on the runner to instead run with fewer iterations.

Two questions:
1. Could this be broken out into its own patch? It seems relatively atomic
2. Does this preserve the nature of the tests -- i.e. do we still hit all the JIT tiers that we expect to, or could the reduced iteration count (in some cases down to 50) prevent that?

Reply:
Ah, these changes were from Justin's original patch. I'm not 100% sure the original reasoning behind reducing the iteration count, but I don't think it hurts to run it the original number of times.

Reply:
I think we try to use 10k since that's usually sufficient to test everything (at least with the eager configurations)

Reply:
Updated tests to all run 10k to be consistent

Reply:
Yeah, I lowered the iteration count so that these tests wouldn't break the infrastructure. 10k seems reasonable.